tests/network-tests/src/flows/storage/initStorage.ts

```diff
@@ -96,7 +114,7 @@ export default function createFlow({ buckets, dynamicBagPolicy }: InitStorageCon
     // Hire operators
     // const hireWorkersFixture = new HireWorkesFixture(api, totalBucketsNum, WorkingGroups.Distribution)
     // await new FixtureRunner(hireWorkersFixture).run()
-    // const operatorIds = hireWorkersFixture.getHiredWorkers()
+    // const operatorIds = hireWorkersFixture.getCreatedWorkerIds()
 
     const operatorIds = buckets.map((b) => createType('WorkerId', b.operatorId))
```

Comment:
Can we remove the commented code here when operator ids seems to be available anyway?